src/layout/Error/Error.scss

```diff
@@ -0,0 +1,123 @@
+@import '@axa-fr/react-toolkit-core/dist/assets/scss/core.scss';
+
+$font-family-serif: Georgia, 'Times New Roman', Times, serif, arial;
+
+.af-container--notfound {
```

Comment:
may add a prefix SASS variable for all starter demo like $prefix: 'af'

Reply:
It's a good thing to have in our todo, I'll create an issue